mods/ts/rules/civilian-structures.yaml

```diff
@@ -1306,6 +1252,7 @@ GASPOT:
 	Buildable:
 		Queue: Defense
 		Prerequisites: ~disabled
+		Description: Civilian Building
```

Comment:
Because I removed it from `^CivBuilding`: https://github.com/OpenRA/OpenRA/pull/12731/commits/39a678d81aad9c10ecaae83df6a300934d3e9762#diff-686e172db95bb90984aec1bc804d5e5eL198
I readded those descriptions to the only two buildable structures to compensate that.